src/main/java/ludum/dare/screens/UpgradeScreen.java

```diff
@@ -87,7 +87,7 @@ public void render(float delta) {
             }
         }
         if (allReady) {
-            game.setScreen(new RaceScreen(game));
+            game.setScreen(new FightScreen(game));
```

Comment:
I left this in so you could try it out, not really different from RaceScreen other than it doesn't do the split screen thing.